src/Config/packages.aget

```diff
@@ -2,12 +2,12 @@
   "nuget": {
     "references": {
       "DynamicLanguageRuntime" : "1.2.1",
-      "DynamoVisualProgramming.Core" : "2.1.0-beta7386",
-      "DynamoVisualProgramming.DynamoCoreNodes" : "2.1.0-beta7386",
-      "DynamoVisualProgramming.DynamoServices" : "2.1.0-beta7386",
-      "DynamoVisualProgramming.Tests" : "2.1.0-beta7386",
-      "DynamoVisualProgramming.WpfUILibrary" : "2.1.0-beta7386",
-      "DynamoVisualProgramming.ZeroTouchLibrary" : "2.1.0-beta7386",
+      "DynamoVisualProgramming.Core" : "2.1.0.7436",
```

Comment:
Is this should be updated to latest version 2.1.0.7451(https://www.nuget.org/packages/DynamoVisualProgramming.Core/2.1.0.7451)?

Reply:
There is no difference other than the samples but sure feel free to do so